src/components/SignupButton.js

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+
+const SignupButtonStyled = styled.button`
+  width: 100px;
+  height: 30px;
+  border-radius: 10px 10px 10px 10px;
+  color: white;
+  opacity: 70%;
```

Comment:
instead of using opacity, maybe you could try rgba colors. According to this article:https://wiki.selfhtml.org/wiki/CSS/Eigenschaften/Anzeige/opacity opacity won´t work if it is written as "%". Nonetheless it is possible that it was changed. 

Is there a reason you don´t work with Themes? It would make it much easier to change the styling later on.